wallet/types/src/account_info.rs

```diff
@@ -110,9 +110,14 @@ pub struct AccountVrfKeys {
 #[derive(Debug, Clone, Encode, Decode)]
 pub enum AccountStandaloneKey {
     #[codec(index = 0)]
-    V0 {
+    Address {
         label: Option<String>,
         public_key: Option<PublicKey>,
```

Comment:
Why not use a destination here?

Reply:
deleted it is already present in the key, this was left from when the key was just PublicKeyHash now it is Destination